helm/macropower-analytics-panel-server/templates/ingress.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ .Values.namespace }}
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  rules:
+  - host: {{ .Values.server.ingress.host }}
+    http:
+      paths:
+      - backend:
+          service:
+            name: {{ include "resource.default.name" . }}
+            port:
+              number: {{ .Values.server.service.port }}
+          path: /{{ .Values.server.ingress.analytics }}(/|$)(.*)
```

Sorry, forgot to mention before, shouldn't this be

```suggestion
          path: /{{ .Values.server.ingress.path }}(/|$)(.*)
```

It's probably fine to only define in `values.yaml` as this should be consistent across all installations as we'll need to refer to that path from dashboard definitions. You know more about the app setup, just thought I'll mention this just in case.